src/sql/src/session/vars.rs

```diff
@@ -621,6 +621,15 @@ pub const ENABLE_RBAC_CHECKS: ServerVar<bool> = ServerVar {
     safe: true,
 };
 
+pub const FORCE_INTROSPECTION_CLUSTER: ServerVar<bool> = ServerVar {
+    name: UncasedStr::new("force_introspection_cluster"),
+    value: &false,
+    description:
+        "Boolean flag indicating whether we should force queries that depend only on system tables, to run on the mz_introspection cluster (Materialize).",
```

Comment:
```suggestion
        "Whether to force queries that depend only on system tables to run on the mz_introspection cluster (Materialize).",
```

(The "Boolean flag indicating..." phrasing is inconsistent with how PostgreSQL phrases its parameters. I know it's what we use for the `emit_`... parameters but we should probably fix that. The type is declared elsewhere so we don't need to mention it in the description itself.)

Reply:
Makes sense! Updated the message to follow this convention